webpack.config.js

```diff
@@ -6,7 +6,10 @@ module.exports = {
   },
   entry: {
     collectionLinks: './src/main/webapp/collectionLinks/index.js',
-    search: './src/main/webapp/search/index.js'
+    search: './src/main/webapp/search/index.js',
+    pages: './src/main/webapp/pages/index.js',
+    consentGroup: './src/main/webapp/consentGroup/index.js'
+
   },
   output: {
     path: path.join(__dirname, 'grails-app/assets/javascripts'),
```

Comment:
One thing I meant to do was to figure out how to update the build process so we don't have to check in the compiled versions of these `index.js` files. The fact that there is a prod/dev version of these can lead to problems if we check in the wrong version. Not for this PR, but we should add a story that does the following:

* Update the build task (war) to run the prod webpack command before building.
* Move all compiled js references to point to a new directory, something like `grails-app/assets/javascripts/compiled`
* Change this output path to match
* Set that directory to be ignored in `.gitignore`

I think that would be enough to get us away from checking in the compiled js code.

Reply:
@rushtong I can take care of that and put it in our "For Refinement" backlog.